reference-architecture/rhv-ansible/ocp-vars.yaml

```diff
@@ -76,8 +82,12 @@ openshift_master_htpasswd_users: {'admin': '$apr1$zAhyA9Ko$rBxBOwAwwtRuuaw8OtCwH
 # or
 #openshift_master_htpasswd_file=<path to local pre-generated htpasswd file>
 
-# NFS for registry storage
+##############################################################################
+### Registry storage
+## NFS
 openshift_hosted_registry_storage_kind: nfs
+openshift_hosted_registry_replicas: 2
```

Comment:
With NFS and the registry you can only have 1 replica.

Reply:
@cooktheryan Thanks for the catch, I pulled that in while trying to go with Gluster.